markdown/events/2022/hackathon-march-2022.md

```diff
@@ -45,6 +45,8 @@ testing each command making sure they work, checking phrasing and descriptions t
 To keep things manageable, we will organise into the groups.
 You are free to change groups as you wish during the hackathon.
 
+For the first time, we will also have talks and presentations during the hackathon.
```

Comment:
Not sure it's really the first time?

Reply:
Ah ok... I didn't remember any non-n-core ones. Sorry! Will remove